### DISCUSSION TITLE
Allow easy customizing of the tray icon

### DISCUSSION THREAD

**Discussion by Geroyuni** [File-New-Project/EarTrumpet#655] [Ideas]
At least the static eartrumpet one, to the image of choice.

I'd also be really grateful if anyone can point out a way in which this is already possible.

**Comment by gaberad** [comment]
Related to this, would it be possible have different icons show based on the selected output? 

e.g. the default speaker icon for speakers, headphone icon for headphones etc

https://github.com/davkean/audio-switcher has a nice headphones icon that fits with the rest of the tray icons

**Comment by JavierSegoviaCordoba** [comment]
Selected device in the tray icon should be great
